src/ucp/api/ucp.h

```diff
@@ -2634,6 +2634,15 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
  *
  * @note The user should not modify any part of the @a buffer after this
  *       operation is called, until the operation completes.
+ * @note The argument @a param->op_attr_mask should be OR combination of
```

Comment:
maybe `The argument @a param->op_attr_mask may be combination of any of the following flags: ....`
i'd also add this phrase here: ` For every UCP_OP_ATTR_FIELD.. flag the corresponding @param structure field must be defined.`
The you will not need to explain param fields separately for every operation